components/Markdown/Markdown.js

```diff
@@ -0,0 +1,15 @@
+import ReactMarkdown from "react-markdown";
+import PropTypes from "prop-types";
+
+const Markdown = ({ children, className = "", ...props }) => (
+  <div className={`prose [&_*]:text-gray-800 ${className}`} {...props}>
```

Comment:
wouldn't something like `text-gray-800!` work as well? Very minor, ignore me, just curious about the selector